README.md

```diff
@@ -183,17 +183,19 @@ const cardSchemes = new Set([
   Fidel.CardScheme.americanExpress
 ]);
 
+const countries = [Fidel.Country.ireland, Fidel.Country.unitedStates];
+
 Fidel.setOptions({
   bannerImage: resolvedImage,
-  country: Fidel.Country.sweden,
+  allowedCountries: countries,
   supportedCardSchemes: Array.from(cardSchemes),
   autoScan: false,
   metaData: {'meta-data-1': 'value1'}, // additional data to pass with the card
   companyName: 'My RN Company', // the company name displayed in the checkbox text
   deleteInstructions: 'Your custom delete instructions!',
   privacyUrl: 'https://fidel.uk',
-  termsConditionsUrl: 'https://fidel.uk/privacy', // mandatory when you set the default country to USA/Canada or when the user can select USA/Canada
-  programName: 'My program name', // optional, is used when you set the default country to USA/Canada or when the user can select USA/Canada
+  termsConditionsUrl: 'https://fidel.uk/privacy', // mandatory when you set the allowed countries to USA/Canada or when the user can select USA/Canada
```

Comment:
the comment here can be improved a bit, to make more sense in English :D. The same for the one below.